jaeger-spark-dependencies-cassandra/src/main/java/io/jaegertracing/spark/dependencies/cassandra/CassandraSpan.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2017 The Jaeger Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.jaegertracing.spark.dependencies.cassandra;
+
+import io.jaegertracing.spark.dependencies.model.Reference;
+import io.jaegertracing.spark.dependencies.model.Span;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Pavol Loffay
+ */
+public class CassandraSpan extends Span {
```

Comment:
I assume this class is being used to deal with the schema changes.

Just wondering if it is worth marking it as deprecated, so possible when moving to jaeger-2.x this could be removed?

Reply:
Not sure. If it will be possible we can make this code compatible with both versions at the same time.

Reply:
Though I will add a comment why there is this class.